bazel-haskell-deps.bzl

```diff
@@ -417,6 +417,7 @@ exports_files(["stack.exe"], visibility = ["//visibility:public"])
         ),
         haddock = False,
         local_snapshot = "//:stack-snapshot.yaml",
+        stack_snapshot_json = "//:stackage_snapshot.json" if not is_windows else None,
```

Comment:
What breaks on Windows?

Reply:
The lock file is generated on Unix and includes Unix specific dependencies, such as `unix`. We would need a separate lock file on Windows, which is going to be inconvenient to generate for most without easy access to a Windows machine.